utils/mkhtml.py

```diff
@@ -305,11 +305,8 @@ def get_addon_path(pgm):
             with open(addons_paths, "r") as f:
                 addons_paths = json.load(f)
             for addon in addons_paths["tree"]:
-                split_path = addon["path"].split("/")
-                root_dir, module_dir = split_path[0], split_path[-1]
-                if "grass8" == root_dir and pgm == module_dir:
-                    return True, addon["path"]
-    return None, None
+                if pgm == addon["path"].split("/")[-1]:
+                    return addon["path"]
```

Comment:
`bool(addon["path"])` to really return True as documentation says. Otherwise, this is usually documented as _a value which converts to True_, but you would be creating potential for wrong use of the API.